src/java/org/apache/cassandra/repair/messages/RepairMessage.java

```diff
@@ -56,8 +60,23 @@
 public abstract class RepairMessage
 {
     private enum ErrorHandling { NONE, TIMEOUT, RETRY }
-    private static final CassandraVersion SUPPORTS_RETRY = new CassandraVersion("5.0.0-alpha2.SNAPSHOT");
+    @VisibleForTesting
+    static final CassandraVersion SUPPORTS_RETRY = new CassandraVersion("5.0.0-alpha2.SNAPSHOT");
     private static final Map<Verb, CassandraVersion> VERB_TIMEOUT_VERSIONS;
+    public static final Set<Verb> ALLOWS_RETRY;
+    private static final Set<Verb> SUPPORTS_RETRY_WITHOUT_VERSION_CHECK = Collections.unmodifiableSet(EnumSet.of(Verb.CLEANUP_MSG));
+    public static final RequestCallback<Object> NOOP_CALLBACK = new RequestCallback<>()
```

Comment:
nit: I think we can move it to the `RequestCallback` interface e.g. 

```
public interface RequestCallback<T>
{
    RequestCallback<Object> NOOP_CALLBACK = msg -> {};
    
    ...
}
```

Reply:
in this case I rather not.  There isn't a reason to use a no-op callback in MessagingService as `null` is what you actually want (noop has higher cost than null...), but since we are doing retries we need a way to tell you we are done, so this should really only be used with retries, so feel its best to stay here